docs/GettingStartedDocs/Contributors/building_oe_sdk.md

```diff
@@ -33,17 +33,22 @@ Please refer to the following [documentation](/docs/GettingStartedDocs/SGXSuppor
    | SGX1                       | SGX1 or SGX1+FLC                    |
    | Simulator                  | Any level                           |
 
-   If your target system does not have any SGX hardware support, you may want to choose "Simulator" mode.
+   On Linux, if your target system does not have any SGX hardware support, you may want to choose "Simulator" mode.
+   On Windows, Open Enclave SDK does not support "Simulator" mode.
 
 #### 3. Build, install and run
 
    Choose an operating mode that is compatible with the SGX support level of your target system.
    The links below contain instructions on how to set up Open Enclave SDK environment for a given mode.
 
+On Linux
   - [Setup Open Enclave SDK for SGX1+FLC mode](SGX1FLCGettingStarted.md)
   - [Setup Open Enclave SDK for SGX1 mode](SGX1GettingStarted.md)
   - [Setup Open Enclave SDK for Simulator mode](SimulatorGettingStarted.md)
 
+On Windows
+ - [Set up Open Enclave SDK](/docs/GettingStartedDocs/GettingStarted.Windows.md)
```

Comment:
We probably want to move this doc into Contributors folder at some point for parity with the Linux ones, but we'll take care of that after the other changes in #2081 and #2097 have gone in.

Reply:
Agree with this. will move it around in next PR.